diff -Naur xfce4-power-manager-1.4.0.orig/settings/xfce4-power-manager-settings.desktop.in xfce4-power-manager-1.4.0/settings/xfce4-power-manager-settings.desktop.in
--- xfce4-power-manager-1.4.0.orig/settings/xfce4-power-manager-settings.desktop.in	2014-05-25 00:59:48.000000000 +0700
+++ xfce4-power-manager-1.4.0/settings/xfce4-power-manager-settings.desktop.in	2014-09-28 21:38:09.181184989 +0700
@@ -8,7 +8,8 @@
 Terminal=false
 Type=Application
 Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-HardwareSettings;
-NotShowIn=GNOME;KDE;Unity;
+#NotShowIn=GNOME;KDE;Unity;
+OnlyShowIn=XFCE;GNOME;
 StartupNotify=true
 X-XfcePluggable=true
 X-XfceHelpComponent=xfce4-power-manager
